_posts/fr/newsletters/2023-01-04-newsletter.md

```diff
@@ -0,0 +1,217 @@
+      postée en 2015 par Peter Todd. Dans une réponse [ultérieure][todd fss],
+      Todd a décrit plusieurs façons dont l'idée est beaucoup moins
+      préférable que l'opt-in ou le RBF complet.
+
+    - *Motivation du Full-RBF :* Anthony Towns [a répondu][towns rbfm] à
+      un fil de discussion sur la motivation de divers groupes à effectuer
+      des full-RBF. Towns analyse ce que la rationalité économique
+      signifie---et ne signifie pas---dans le contexte de la sélection des
+      transactions des mineurs. Les mineurs optimisant leurs profits à très
+      court terme préféreraient naturellement le full-RBF. Cependant, Towns
+      note que les mineurs qui ont fait un investissement à long terme dans
+      l'équipement minier pourraient plutôt préférer optimiser les revenus
+      des frais sur plusieurs blocs, et cela pourrait ne pas toujours favoriser
+      le full-RBF. Il suggère trois scénarios possibles à considérer.
+
+## Mises à jour et version candidate
+
+*Nouvelles versions et versions candidates pour les principaux projets d'infrastructure Bitcoin.
+Veuillez envisager de passer aux nouvelles versions ou d'aider à tester les versions candidates.*
+
+- [Eclair 0.8.0][] est une version majeure de cette implémentation
+  populaire de nœuds LN. Elle ajoute la prise en charge des [canaux zéro-conf][topic zero-conf channels]
+  et des alias SCID (Short Channel IDentifier). Consultez ses [notes de version][eclair 0.8 rn]
+  pour plus d'informations sur ces fonctionnalités et d'autres changements.
+
+- [LDK 0.0.113][] est une nouvelle version de cette bibliothèque
+  permettant de créer des portefeuilles et des applications compatibles
+  avec LN.
+
+- [BDK 0.26.0-rc.2][] est une version candidate de cette bibliothèque
+  pour la création de portefeuilles.
+
+## Changements principaux dans le code et la documentation
+
+*Changements notables cette semaine dans [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], et [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #26265][] relaxes the minimum permitted non-witness
+  serialized size of transactions in transaction relay policy from 82
+bytes to 65 bytes. For example, a transaction with a single input and
+single output with 4 bytes of OP\_RETURN padding, which previously
+would have been rejected for being too small, could now be accepted
+into the node's mempool and relayed. See [Newsletter #222][min relay
+size ml] for background information and motivation for this change.
+
+- [Bitcoin Core #21576][] allows wallets using an external signer (e.g. [HWI][topic hwi]) to fee bump
+  using [opt-in RBF][topic rbf] in the GUI and when using the `bumpfee` RPC.
```

Comment:
en utilisant l'[opt-in RBF][topic rbf] dans l'interface graphique et lors de l'utilisation du RPC `bumpfee`.